deployment/TESTING.md

```diff
@@ -0,0 +1,24 @@
+# POA Token Bridge / Deployment Testing
+
+The deployment playbooks are tested using [Molecule](https://molecule.readthedocs.io).
+
+## Prepare virtual python environment
+
+```
+command -v virtualenv || pip3 install virtualenv
```

Comment:
does it mean that to test deployment locally I need to install python3, virtualenv and other things listed in `requirements.txt`? Is it possible to containerize this by a docker file as so an adapter could test the deployment scripts without necessity to encumber the system with additional packages? It will definitely simplify the work process.

Reply:
The packages listed in `requirements.txt` are not polluting the system but are installed in a folder much like `node_modules` in JS - that's thanks for `virtualenv`.

But yes, you still need `python3` and `virtualenv` installed.

I created #118 for this.

Reply:
Thanks! I think we cannot finish the phase 2 without addressing #118.